.gitignore

```diff
@@ -43,6 +43,7 @@ terraform/.terraform/*
 .env
 .hypothesis/
 
+*.rst
```

Comment:
As discussed in other PRs, we'll want to check in some `.rst` files into git, so this should be removed.